_posts/en/pages/2016-01-01-about.md

```diff
@@ -5,15 +5,35 @@ permalink: /en/about/
 type: pages
 layout: page
 lang: en
-version: 1
+version: 2
 redirect_from:
-  - /zh_TW/about/
```

Comment:
Why remove this redirect?

Reply:
I moved it to `data.navigation.zh_TW`, like all other redirects.

Reply:
No you haven't. Check https://bitcoincore.org/zh_TW/about/ on the live site and on this PR:

- The live site redirects to https://bitcoincore.org/en/about/;
- this PR gives a 'Page not found' error.

Reply:
I don't think this matters much, since no one should have that page bookmarked. They click on it in the navigation and get redirected to the en site, so not really a way to copy paste the link. Be reminded that the only use of this link (the sponsorship page) has already been removed.

Reply:
Ok, I was wrong a second time. This could have ended up in someones bookmarks before ef1e936e13604960a4cb8c14095de1be5619de3b. Though, I am still not convinced that adding the redirect here is worth it.

Reply:
No need to add it. Just don't remove it :)

Reply:
Done